src/components/Form.js

```diff
@@ -1,9 +1,26 @@
-const Form = () => (
-  <form className="form">
-    <input type="text" name="author" placeholder="Author" className="author" />
-    <input type="text" name="book" placeholder="Book" className="book" />
-    <input type="submit" value="Submit" className="submit" />
-  </form>
-);
+import { useDispatch } from 'react-redux/es/exports';
+import React, { useRef } from 'react';
+import uniqid from 'uniqid';
+import { addBook } from '../redux/books/books';
+
+const Form = () => {
+  const dispatch = useDispatch();
+  const title = useRef(null);
+  const author = useRef(null);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    dispatch(addBook(title.current.value, author.current.value, uniqid()));
+    e.target.reset();
```

Comment:
- [ ] You have demonstrated an understanding of the `useRef` hook which is good 👏 However, for the purpose of this milestone, you are required to have a local state with `title` and `author` and they should have their inputs that updates the local state every time they update (`onChange`)